src/main/java/appeng/fluids/contents/FluidCellConfig.java

```diff
@@ -91,16 +74,12 @@ public boolean isItemValidForSlot( int slot, ItemStack stack )
 		{
 			super.isItemValidForSlot( slot, stack );
 		}
-		FluidStack fluid = FluidUtil.getFluidContained( stack );
-		if( fluid == null || !Api.INSTANCE.definitions().items().dummyFluidItem().maybeStack( 1 ).isPresent() )
+		ItemStack fs = FluidDummyItem.createStackfromFluidContainer( stack );
+		if (fs.isEmpty())
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') '(' is not followed by whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') ')' is not preceded with whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)